### PR TITLE
failing test cases errors resolved

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,7 +11,6 @@ Dockerfile
 
 # test_files
 dry_run.py
-test_*
```
Comment (ashwin1111) [inline .gitignore:14]: let's not remove this, kept it to ignore test scripts

File: test/integration/data/expense_reports/tstdrv2089588.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
         "type": "vendor"
     },
     "expenseReportCurrency": {
-        "name": "USD",
+        "name": "USA",
```
Comment (ashwin1111) [inline test/integration/data/expense_reports/tstdrv2089588.json:20]: did this actually work? how can USA be a currency, do we have one in our account?

File: test/integration/test_accounts.py [hunk 3]
```diff
@@ -15,11 +15,14 @@ def test_get(nc):
 
 
 def test_get_all_generator(nc):
-    res1 = nc.accounts.get_all()
-    res2 = []
+    get_all_response = nc.accounts.get_all()
+    get_all_genrator_respose = []
```
Comment (ashwin1111) [inline test/integration/test_accounts.py:19]: `get_all_generator_response` typo

File: test/integration/test_subsidiaries.py [hunk 4]
```diff
@@ -14,11 +14,14 @@ def test_get(nc):
...
+    get_all_respose = nc.subsidiaries.get_all()
+    get_all_generator_response = []
     for r in nc.subsidiaries.get_all_generator(page_size=200):
-        res2.append(r)
-    assert len(res1) == len(res2), 'changing page size is returning different results'
+        get_all_generator_response.append(r)
+    len_get_all_generator_response = 0
+    for i in get_all_generator_response:
+        len_get_all_generator_response += len(i)
+    assert len(get_all_respose) == get_all_generator_response, 'changing page size is returning different results'
```
Comment (ashwin1111) [inline test/integration/test_subsidiaries.py:24]: compare with `len_get_all_generator_response`

File: test/integration/data/expense_reports/tstdrv2089588.json (hunk 2 above)
Comment (ruuushhh) [reply test/integration/data/expense_reports/tstdrv2089588.json:20]: We are using this in netsuite as well as in fixtures

File: .gitignore (hunk 1 above)
Comment (ruuushhh) [reply .gitignore:14]: It skips test files for my branch as well, will update it once we are done with all other changes

File: test/integration/test_subsidiaries.py [hunk 5]
```diff
@@ -14,11 +14,14 @@ def test_get(nc):
     assert data, f'No object with internalId {internal_id}'
 
 def test_get_all_generator(nc):
-    res1 = nc.subsidiaries.get_all()
-    res2 = []
+    get_all_respose = nc.subsidiaries.get_all()
```
Comment (ashwin1111) [inline test/integration/test_subsidiaries.py:17]: `get_all_response` typo

File: test/internal/test_logout.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+import os
+import time
+
+import pytest
+from netsuitesdk.internal.client import NetSuiteClient
+from netsuitesdk.internal.utils import PaginatedSearch
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.skip(reason="We will not use this often")
```
Comment (ashwin1111) [inline test/internal/test_logout.py:11]: if this is working fine without any error, we shouldn't skip this

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**